Cargo.lock

```diff
@@ -173,9 +173,9 @@ checksum = "d468802bab17cbc0cc575e9b053f41e72aa36bfa6b7f55e3529ffa43161b97fa"
 
 [[package]]
 name = "axum"
-version = "0.6.11"
+version = "0.6.15"
```

Comment:
We pin this to 0.6.11 in `Cargo.toml`, shouldn't we change it there rather than in `Cargo.lock`?

https://github.com/getsentry/relay/blob/751bc4b2f4e3c6fc7373d8ca7cddf0812f68dbbc/relay-server/Cargo.toml#L31